test/uk/gov/hmrc/merchandiseinbaggage/service/EmailServiceSpec.scala

```diff
@@ -17,36 +17,56 @@
 package uk.gov.hmrc.merchandiseinbaggage.service
 
 import org.scalamock.scalatest.MockFactory
+import org.scalatest.OptionValues
 import org.scalatest.concurrent.ScalaFutures
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.merchandiseinbaggage.connectors.EmailConnector
 import uk.gov.hmrc.merchandiseinbaggage.model.DeclarationEmailInfo
 import uk.gov.hmrc.merchandiseinbaggage.model.api.Declaration
+import uk.gov.hmrc.merchandiseinbaggage.model.api.DeclarationType.Export
 import uk.gov.hmrc.merchandiseinbaggage.repositories.DeclarationRepository
 import uk.gov.hmrc.merchandiseinbaggage.{BaseSpecWithApplication, CoreTestData}
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.{ExecutionContext, Future}
 
-class EmailServiceSpec extends BaseSpecWithApplication with CoreTestData with ScalaFutures with MockFactory {
+class EmailServiceSpec extends BaseSpecWithApplication with CoreTestData with ScalaFutures with MockFactory with OptionValues {
 
   val declarationRepo = mock[DeclarationRepository]
   val emailConnector = mock[EmailConnector]
 
   val emailService = new EmailService(emailConnector, declarationRepo)
 
-  "sendEmails" when {
-    "sending emails to new declarations" should {
-      "send" in {}
-    }
+  "sendEmails should handle New Import declarations" in {
```

Comment:
not sure if import&export is worth to be combined in to a single test by using `forAll(declarationTypesTable) { importOrExport: DeclarationType =>`

Reply:
yeah, combining into one test , may make the test difficult sometimes as we have to do if else checks for verifications... but dont mind really, i am happy to do it in a single test...